app/policies/user_policy.rb

```diff
@@ -20,7 +20,8 @@ class UserPolicy < ApplicationPolicy
     email_follower_notifications
     email_membership_newsletter
     email_mention_notifications
-    email_newsletter email_public
+    email_newsletter
+    email_public
```

Comment:
This is now on `Users::Setting` and called `display_email_on_profile`. It's probably best to wait for Arit's PR to be merged then rebase this against `main`.